app/assets/stylesheets/components/_header-notice.scss

```diff
@@ -51,3 +51,7 @@
 .app-c-header-notice__call-to-action {
   @include govuk-font(16, $weight: bold);
 }
+
+.app-c-header-notice__call-to-action-description {
+  @include govuk-font(16);
```

Comment:
If you like, this could be removed completely and in the markup replaced with `govuk-body-s` instead? (https://design-system.service.gov.uk/styles/typography/#body-small)